farm/modeling/prediction_head.py

```diff
@@ -1503,7 +1508,6 @@ def merge_formatted_preds(preds_all):
 
         # This fn is used to align QA output of len=n_docs and Classification output of len=n_passages
         def chunk(iterable, lengths):
-            assert sum(lengths) == len(iterable)
```

Comment:
I do not know what this assert does... lets discuss in detail or throw a logger.error?